frontend/src/queries/schema.ts

```diff
@@ -699,3 +699,10 @@ export interface BreakdownFilter {
     breakdown_group_type_index?: number | null
     breakdown_histogram_bin_count?: number // trends breakdown histogram bin count
 }
+
+export interface DashboardFilter {
+    date_from?: string | null
+    date_to?: string | null
+    properties?: AnyPropertyFilter[] | null
+    [s: string]: any
```

Comment:
I wonder if this `[s: string]: any` line is even needed? Might be well to strictly scope down the dashboard filters to just dates and properties.

Reply:
I thought this would be tricky due to other types depending on this, but actually was quite simple and is now done